components/ErrorBanner/OldDesignatedVotingAccountWarningBanner.tsx

```diff
@@ -0,0 +1,79 @@
+import { useStakingContext } from "hooks";
+import NextLink from "next/link";
+import styled from "styled-components";
+import { useIsClient, useLocalStorage } from "usehooks-ts";
+import {
+  CloseButton,
+  CloseIcon,
+  ErrorMessage,
+  ErrorMessageWrapper,
+  IconWrapper,
+  WarningIcon,
+  Wrapper,
+} from "./styles";
+
+export function OldDesignatedVotingAccountWarningBanner() {
+  const {
+    oldDesignatedVotingAccount: {
+      isOldDesignatedVotingAccount,
+      message,
+      designatedVotingContract,
+    },
+  } = useStakingContext();
+  const [closeClicked, setClosedClicked] = useLocalStorage(
+    "show-old-designated-voting-account-warning-banner",
+    false
+  );
+
+  const isClient = useIsClient();
+
+  const show = isOldDesignatedVotingAccount && !closeClicked;
+
+  if (!isClient || !show) return null;
+
+  function close() {
+    setClosedClicked(true);
+  }
+
+  return (
+    <Wrapper>
+      <ErrorMessageWrapper>
+        <IconWrapper>
+          <WarningIcon />
+        </IconWrapper>
+        <ErrorMessage>
+          {message}:{" "}
+          <Link
+            href={`https://etherscan.io/contract/${designatedVotingContract}`}
```

Comment:
this doesnt work for some reason. sends me to a 404. also does this matter if its on goerli?

Reply:
I used `contract` in the url but should have used `address`. Pushed a fix now. 

afaik this does not apply on goerli. @chrismaree is that correct?